### PR TITLE
[MW] JSS Uptime Suggestion / Checklist / Stats

### PR DESCRIPTION
**Pull request by anom0ly**
Added new information on JSS soothing mist uptime and corresponding statistics, checklist item, and suggestion..

image

### CODE REVIEW

**Comment by yajinni** [comment]
if this is a talent shouldnt it use the talent box and not statistic box?

**Comment by anom0ly** [comment]
Interesting - I honestly didn't know we had added that. Let me update not only this new talent statistic, but also the others, which I still using just the plain statisticbox component.

**Comment by anom0ly** [comment]
Updated screenshot:
image

File: src/parser/monk/mistweaver/modules/talents/SpiritOfTheCrane.js [hunk 1]
```diff
@@ -132,30 +132,25 @@ class SpiritOfTheCrane extends Analyzer {
...
-          >
-            Mana Returned
-          </dfn>
-        )}
+        label="Mana Returned"
+        tooltip={`
+          You gained a raw total of ${((this.manaReturnSotc + this.sotcWasted) / 1000).toFixed(0)}k mana from SotC with ${(this.sotcWasted / 1000).toFixed(0)}k wasted.<br>
+          You lost ${(this.totmOverCap + this.totmBuffWasted)} Teachings of the Monestery stacks
+          <ul>
+          ${this.totmOverCap > 0 ?
```
Comment (MartijnHols) [inline src/parser/monk/mistweaver/modules/talents/SpiritOfTheCrane.js]: For future reference the indentation here gets a bit weird, but I'll fix it when I merge this into `next`.

**Pull request merged**